tests/e2e/timeline/timeline.spec.js

```diff
@@ -72,7 +72,7 @@ test.describe('Timeline spec', () => {
       })
     })
 
-    await page.goto('/projects?p=!C1443528505-LAADS&sb=-77.15071678161621%2C38.78817179999825%2C-76.89801406860352%2C38.99784152603538&lat=37.64643450971326&long=-77.407470703125&zoom=7qt=2020-01-06T04%3A15%3A27.310Z%2C2020-01-13T07%3A32%3A50.962Z&ff=Map%20Imagery&tl=1563377338!4!!')
+    await page.goto('/projects?p=!C1443528505-LAADS&sb=-77.15071%2C38.78817%2C-76.89801%2C38.99784&lat=37.64643&long=-77.40747&zoom=7qt=2020-01-06T04%3A15%3A27.310Z%2C2020-01-13T07%3A32%3A50.962Z&ff=Map%20Imagery&tl=1563377338!4!!')
```

Comment:
can we figure out why this happend?

Reply:
I rolled react-datetime back to   `"react-datetime": "^2.16.3",` and the test still passed. not completely sure whats going on here but it doesn't appear to be harmful

Reply:
I don't think we need to rollback react-datetime, these numbers are spatial drawings, or map lat/long

Reply:
Oh wait, these values are in a page.goto, so it is just what we are putting into the URL bar to load the page. I really don't know why they changed